hbase-server/src/main/java/org/apache/hadoop/hbase/TableDescriptors.java

```diff
@@ -31,7 +31,7 @@
   /**
    * @return TableDescriptor for tablename
    */
-  TableDescriptor get(final TableName tableName) throws IOException;
+  TableDescriptor get(TableName tableName) throws IOException;
```

Comment:
Is this change needed? Do we follow any convention to not provide `final` to argument? Just asking as I am not aware of it.

Reply:
In the interface the final is useless, you are free to not use final when actually implementing the method, and there is no difference for callers. So just remove it.